app5/task11.cs

```diff
@@ -0,0 +1,41 @@
+using functions;
+
+namespace task11
+{
+    public class BinarySearch
+    {
+        public static void Result()
+        {
+            int numberN = Functions.CanInt();
+            int[] elementsArr = Functions.InsertIntArr(numberN);
+            int numToFind = Functions.CanInt();
+
+            int left = 0;
+            int right = elementsArr.Length - 1;
+            bool isInside = false;
+            int index = 0;
+
+            while (left <= right)
+            {
+                int mid = left + (right - left) / 2;
```

Comment:
why not elementsArr.Length / 2; and in math ceil or floor

Reply:
is not an elementsArr.Length / 2   because the task is for binary search and when you always get the center of the array, the algorithm doesn't work.

Reply:
both left and right are integers, if the division is not exact it will be rounded